src/Game.java

```diff
@@ -0,0 +1,149 @@
+
+public class Game {
+    private static final Random RANDOM = new Random();
+
+    private final static int PLAY = 1;
+    private final static int EXIT = 2;
+
+    Scanner scanner = new Scanner(System.in);
+
+    private static final int MAX_ERRORS = 8;
+    private String wordToFind;
+    private char[] wordFound;
+    private int nbErrors;
+    private ArrayList<String> letters = new ArrayList<>();
+    private ArrayList<String> lettersFromFile = new ArrayList<>();
+
+    public void hangman(){
+    boolean isException = true;
+        while (isException) {
+        try {
+            int option = -1;
+            while (option != EXIT) {
+
+                System.out.println("Wybierz opcję:");
+                System.out.println("1 - graj");
+                System.out.println("2 - koniec programu");
+                option = scanner.nextInt();
+                scanner.nextLine();
+
+                switch (option) {
+                    case EXIT:
+                        System.out.println("Bye bye!");
+                        scanner.close();
+                        break;
+                    case PLAY:
+                        play();
+                        break;
+                    default:
+                        System.out.println("Wybrałeś nieprawidłową opcję");
+                }
+            }
+            isException = false;
+        } catch (InputMismatchException e) {
+            System.out.println("Podałeś niepoprawny znak.");
+            scanner.nextLine();
+        }
+    }
+}
+
+    private String nextWordToFind() {
+        return Keywords.WORDS[RANDOM.nextInt(Keywords.WORDS.length)];
+    }
+
+    private void addWord(String word) {
+        lettersFromFile.add(word);
+    }
+
+    private String wordToFindFromFile() {
+        String fileName = "words.txt";
+        try (
+                var fileReader = new FileReader(fileName);
+                var bufferedReader = new BufferedReader(fileReader);
+        ) {
+            while (bufferedReader.readLine() != null) {
+                addWord(bufferedReader.readLine());
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return lettersFromFile.get(RANDOM.nextInt(lettersFromFile.size())).toUpperCase();
+    }
+
+    private void newGame() {
+        nbErrors = 0;
+        letters.clear();
+        wordToFind = nextWordToFind();
+        wordFound = new char[wordToFind.length()];
+
+        Arrays.fill(wordFound, '*');
+    }
+
+    private boolean wordFound() {
+        return wordToFind.contentEquals(new String(wordFound));
+    }
+
+    private String wordFoundContent() {
+        StringBuilder sb = new StringBuilder();
+
+        for (int i = 0; i < wordFound.length; i++) {
+            sb.append(wordFound[i]);
+
+            if (i < wordFound.length - 1) {
+                sb.append(" ");
+            }
+        }
+        return sb.toString();
+    }
+
+    private void enter(String c){
```

Comment:
Why are you using String for just one letter?

Reply:
Because using the IndexOf method of the String class and the while loop we can check all occurrences of a given letter in the searched word

Reply:
I would accept char as param and convert it to a string in a method. I mean, this code should only work with letter and you wrote it so it can accept strings (multiple letters) even though it should not.